pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/impl/NoDictionaryOnHeapDictionaryJointRule.java

```diff
@@ -85,7 +85,7 @@ public void run() {
     noDictCols.removeAll(_output.getIndexConfig().getInvertedIndexColumns());
     noDictCols.remove(_output.getIndexConfig().getSortedColumn());
     noDictCols.removeAll(_output.getIndexConfig()
-        .getRangeIndexColumns()); // TODO: Remove this after range index is implemented for no-dictionary
+        .getRangeIndexColumns());
```

Comment:
nit: merge this line to the previous line?

Reply:
updated!